kratos/python/add_container_variable_data_to_python.cpp

```diff
@@ -0,0 +1,243 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+#include <numeric>
+
+// External includes
+#include <pybind11/stl.h>
+#include <pybind11/operators.h>
+#include <pybind11/numpy.h>
+
+// Project includes
+#include "containers/container_variable_data/container_data_io.h"
+#include "containers/container_variable_data/container_variable_data.h"
+#include "containers/container_variable_data/specialized_container_variable_data.h"
+
+// Include base h
+#include "add_container_variable_data_to_python.h"
+
+namespace Kratos::Python
+{
+
+template <class TDataType>
+using ContiguousNumpyArray = pybind11::array_t<
+    TDataType,
+    /*column-major*/ pybind11::array::c_style | /*cast instead of looking for a suitable overload*/ pybind11::array::forcecast
```

Comment:
this is a leftover, just remove the forcecast tag because we don't actually want that.